.github/workflows/image.yaml

```diff
@@ -0,0 +1,79 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Where do the artifacts go when this is invoked on the master branch?

I wouldn't mind adding the `v*` branches as well (not just the tags)

Reply:
I added the branches, removed the tags. Is that alright?

Reply:
Added tags too.